sorts/stalin_sort.py

```diff
@@ -0,0 +1,37 @@
+"""
+This is a pure Python implementation of the stalin sort algorithm,
+inspired by the dictator of the former soviet union Josef Stalin.
+
+This sorting algorithm sorts a collection of elements by deleting
+every element that is not in order resulting in a sorted collection.
+
+For manual testing run:
+python stalin_sort.py
+"""
+
+
+def stalin_sort(collection):
+    """A pure Python implementation of stalin sort algorithm
+
+    :param collection: a mutable collection of comparable items
+    :return: the same collection ordered by ascending
+
+    Examples:
+    >>> stalin_sort([1, 6, 3, 20, 30])
+    [1, 6, 20, 30]
+    >>> stalin_sort([])
+    []
```

Comment:
This test is failing because the function tries to get the first element of the list when there is none